client/src/main/kotlin/io/provenance/client/grpc/AbstractPbClient.kt

```diff
@@ -168,13 +183,13 @@ open class AbstractPbClient<T : ManagedChannelBuilder<T>>(
     fun estimateAndBroadcastTx(
         txBody: TxOuterClass.TxBody,
         signers: List<BaseReqSigner>,
-        mode: ServiceOuterClass.BroadcastMode = ServiceOuterClass.BroadcastMode.BROADCAST_MODE_SYNC,
+        mode: BroadcastMode = BroadcastMode.BROADCAST_MODE_SYNC,
```

Comment:
Oh, also, I looked into it and it appears that the enum value is left in the proto and marked as deprecated with no signs of plans to remove entirely (though it will blow up in 0.47.x), so that is why I'm not doing some sort of wrapper  mode and sticking with the proto enum for now